libafl/src/statistics/manager/global_stats.rs

```diff
@@ -0,0 +1,173 @@
+//! Global statistics available for Monitors to use
+
+use alloc::string::{String, ToString};
+use core::{cmp, time::Duration};
+
+use libafl_bolts::{current_time, format_duration_hms};
+#[cfg(feature = "std")]
+use serde_json::Value;
+
+use super::ClientStatsManager;
+use crate::statistics::ProcessTiming;
+#[cfg(feature = "std")]
+use crate::statistics::{
+    user_stats::{AggregatorOps, UserStats, UserStatsValue},
+    ItemGeometry,
+};
+
+impl ClientStatsManager {
```

Comment:
It feels odd to me to implement methods on a struct here that's in a different file, it would probably be better to keep them at the same place

Reply:
Yes, I like the idea, but I still think it's better to merge them, simply because we split impls nowhere else in the codebase. It'd be a bit confusing